plugins/plugin-git/che-plugin-git-ext-git/src/main/java/org/eclipse/che/ide/ext/git/client/commit/CommitPresenter.java

```diff
@@ -76,7 +71,7 @@
     private final ProcessesPanelPresenter consolesPanelPresenter;
 
     private Project      project;
-    private Set<String>  allFiles;
+    private List<String> allFiles;
```

Comment:
No, we needn't, but git returns modified filed in some order, so I just do not want to loose it.